src/popover.jsx

```diff
@@ -1,12 +1,24 @@
-import React from 'react'
+import React, { PropTypes, Component } from "react"
+import ReactDOM from "react-dom"
+
+export default class Popover extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    closeOnOuterClick: PropTypes.bool,
+    horizontalJustify: PropTypes.oneOf(["left", "right"]),
+    isOpen: PropTypes.bool,
+    leftOffset: React.PropTypes.number,
+    position: PropTypes.oneOf(["top", "bottom", "left", "right"]),
+    toggleButton: PropTypes.node.isRequired,
```

Comment:
This button prop is not required. Down below we're checking for the presence of it and rendering a default button if it is not provided.